tests/main/services-start/task.yaml

```diff
@@ -41,8 +41,20 @@ execute: |
         systemctl status snap.test-snapd-service-restart.svc${id}.service | MATCH "$EXPECTED_STATUS"
     done
 
+    # When running the test with the --enable option in EXTRA_FLAGS, services 3
+    # and 4 would have been enabled by the
+    #
+    #   snap start $EXTRA_FLAGS test-snapd-service-restart
+    #
+    # command issued before. To make the following step meaningful in that
+    # scenario as well, we now explicitly stop these two services.
+    echo "Stop services 3 and 4"
+    snap stop test-snapd-service-restart.svc3
+    snap stop test-snapd-service-restart.svc4
```

Comment:
That's fine, but I was suggesting to stop just *one* of them, so we would basically be testing two cases :)

Reply:
Ah, indeed! Done.